lib/mozart_fetcher/component.ex

```diff
@@ -61,24 +61,27 @@ defmodule MozartFetcher.Component do
     %{index: component_index, id: id, status: 500, data: %{}}
   end
 
-  defp metric(id, 200) do
+  defp metric(id, _endpoint, 200) do
     ExMetrics.increment("success.component.process")
     ExMetrics.increment("success.component.process.#{id}.200")
   end
 
-  defp metric(id, status) when is_integer(status) do
+  defp metric(id, endpoint, status) when is_integer(status) do
     ExMetrics.increment("error.component.process")
     ExMetrics.increment("error.component.process.#{id}")
     ExMetrics.increment("error.component.process.#{id}.#{status}")
     ExMetrics.increment("error.component.process.#{status}")
-    Stump.log(:error, %{message: "Non-200 response. Got status:#{status} for component #{id}"})
+    Stump.log(:error, %{message: "Non-200 response",
+                        status: status,
+                        component: id,
+                        endpoint: endpoint})
```

Comment:
just double checking that we never fetch from endpoints that have the `?api_key=` query string value? I know we do for context calls, so I'm just checking that isn't the same here so we don't log api keys

Reply:
Good point! I can't think at anything but could happen in the future....

Reply:
At the moment we don't have any api keys for components. We need to log query strings as we currently do this in Requester and this shows the component timeout or query parameters we pass to brokers. If we don't have this, it would be hard to debug any request errors.

Reply:
ok I was already working on it but makes sense, let's keep the qs for now in the logs